### PR TITLE
DOC: List of Features

### PR DESCRIPTION
**Pull request by Lucia-Fonseca**
## Description
This PR adds a new page "Main Functionalities" to the SkyPy documentation with the full list of features in the SkyPy library. 
It will contain all packages in SkyPy (galaxies, utils, and pipeline) and use "astronomer-friendly" language to describe the feature.

Merging this PR partially addresses #443 linked to https://github.com/openjournals/joss-reviews/issues/3056.

## Checklist
- [x] Follow the [Contributor Guidelines](https://github.com/skypyproject/skypy/blob/main/CONTRIBUTING.rst)
- [ ] Write unit tests
- [x] Write documentation strings
- [x] Assign someone from your working team to review this pull request
- [x] Assign someone from the infrastructure team to review this pull request

### CODE REVIEW

**Comment by Lucia-Fonseca** [comment]
Help: the `skypy.utils.random` link is not properly pointing to the right direction.

Q: does it need more astrophysical explanation? A small paragraph, perhaps?

File: docs/main_functionalities.rst [hunk 1]
```diff
@@ -0,0 +1,40 @@
+####################
+Main functionalities
```
Comment (rrjbca) [inline docs/main_functionalities.rst:2]:
```suggestion
Main Features
```
Suggest replacing "functionalities" with "features" everywhere, including renaming this file

File: docs/main_functionalities.rst [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+.. _Luminosity Distributions: https://skypy.readthedocs.io/en/latest/galaxies.html#module-skypy.galaxies.luminosity
+.. _Morphology Distributions: https://skypy.readthedocs.io/en/latest/galaxies.html#module-skypy.galaxies.morphology
+.. _Redshift Distributions: https://skypy.readthedocs.io/en/latest/galaxies.html#module-skypy.galaxies.redshift
+.. _Spectrum Distributions: https://skypy.readthedocs.io/en/latest/galaxies.html#module-skypy.galaxies.spectrum
+.. _Stellar Mass Distributions: https://skypy.readthedocs.io/en/latest/galaxies.html#module-skypy.galaxies.stellar_mass
+
+Utils
+-----
+
+- `Photometry`_ : spectral energy distributions and photometric properties.
```
Comment (rrjbca) [inline docs/main_functionalities.rst:25]:
```suggestion
- `Photometric Calculations`_
```

File: docs/main_functionalities.rst [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+This page outlines the main functionalities in the `SkyPy` library.
+
+Galaxies
+--------
+
+- `Luminosity Distributions`_
+- `Morphology Distributions`_
+- `Redshift Distributions`_
+- `Spectrum Distributions`_
```
Comment (rrjbca) [inline docs/main_functionalities.rst:13]:
```suggestion
- `Spectral Energy Distribution Modelling`_
```

File: docs/main_functionalities.rst [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+Main functionalities
+####################
+
+This page outlines the main functionalities in the `SkyPy` library.
+
+Galaxies
+--------
+
+- `Luminosity Distributions`_
+- `Morphology Distributions`_
```
Comment (rrjbca) [inline docs/main_functionalities.rst:11]:
```suggestion
- `Morphological Distributions`_
```

File: docs/main_functionalities.rst [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+.. _Special Functions: https://skypy.readthedocs.io/en/latest/utils/index.html#module-skypy.utils.special
+
+
+Pipeline
+--------
+- `Pipeline`_:
+    - Running `SkyPy` from the command line
+    - Using a pipeline from other code
+
+.. _Pipeline: https://skypy.readthedocs.io/en/latest/pipeline/index.html
```
Comment (rrjbca) [inline docs/main_functionalities.rst:40]: Is this section necessary? Does it count as part of the "library"?

File: docs/feature_list.rst [hunk 6]
```diff
@@ -0,0 +1,20 @@
+#############
+Feature list
```
Comment (rrjbca) [inline docs/feature_list.rst:2]:
```suggestion
Feature List
```

**Review by rrjbca — APPROVED** [review]

**Pull request merged**